### ISSUE TITLE
Mounting google drive with permition to execut binaries 

### ISSUE THREAD

**Issue by Majdoddin** [googlecolab/colabtools#3162]
**Is your feature request related to a problem? Please describe.**
It seems that Drive is mounted without permission to execute binaries. 
I find no ways in the API to mount Drive with that permission. 

**Describe the solution you'd like**
Something like `exec` option of `mount` in Linux

**Additional context**
This hinders using `venv` of Python effectively. I have used `venv` to create a virtual environment on Drive. 
But next time I mount Drive, I get 
#venv/bin/pip
-bash: venv/bin/pip: /content/drive/MyDrive/ai/whisperscript/venv/bin/python3: bad interpreter: Permission denied
# ls -als venv/bin/pip
1 -rw------- 1 root root 261 Oct 17 11:59 venv/bin/pip

**Comment by Majdoddin** [comment]
@mco-gh Thank you for prompt answer. 

> But note that execute permission doesn't appear to persist across sessions

True, and therefore I made this feature request :) 
This is necessary to make use of python venv across sessions, which would save a lot of time installing packages, and it is not practical to run `chmod ` on all scripts each time. 

> I suspect it's an artifact of the locally mounted image of your Drive resources, not stored natively in Drive

You mean like an overlay filesystem? But how is it that the files I create in mounted Drive are immediately accessible in Drive web interface?

```
#cat /proc/mounts
drive /content/drive fuse.drive rw,nosuid,nodev,relatime,user_id=0,group_id=0 0 0
```

you see that `noexec` option is not in the output, therefore it is curious why execute permissions are gone.

**Comment by mco-gh** [comment]
> This is necessary to make use of python venv across sessions, which would save a lot of time installing packages, and it is not practical to run chmod on all scripts each time.

Agreed, it's not the most convenient model, but at least it's automatable. You have a code cell that you run to mount Drive, so you can could run a command right after the mount in the same cell that chmod's the executables you care about.  I get that might be a long list and a pain to assemble but once you've got it, it runs automatically every time you mount drive. I'm suggesting this as a workaround, not the ultimate solution.

> You mean like an overlay filesystem? But how is it that the files I create in mounted Drive are immediately accessible in Drive web interface?

What I mean is that the notion of an executable file may not be native to Drive. It may be that the FUSE file system adds executable semantics outside the scope of Drive, which could be why those settings aren't propagated back to Drive. I'm speculating here but that seems plausible to me.

**Comment by Majdoddin** [comment]
I see, thank you very much.
